cytominer_database/ingest.py

```diff
@@ -168,9 +167,5 @@ def seed(source, target, config_path, skip_image_prefix=True):
         for compartment in compartments:
             name, _ = os.path.splitext(os.path.basename(compartment))
 
-            into(
-                input=compartment,
-                output=target,
-                name=name.capitalize(),
-                identifier=identifier,
-            )
+            into(input=compartment, output=target, name=name.capitalize(), identifier=identifier, con=con)
```

Comment:
this too?

Reply:
Thanks - see [comment](https://github.com/cytomining/cytominer-database/pull/135#discussion_r1795994038).